web/src/dnd/onCustomButton.ts

```diff
@@ -0,0 +1,5 @@
+import { fetchNui } from '../utils/fetchNui';
+
+export const onCustomButton = (id: number, slot: number) => {
+  fetchNui('useButton', { id, slot });
+};
```

Comment:
Probably don't really need a separate file for this since it really wont be used outside the context menu.